CMakeLists.txt

```diff
@@ -581,12 +581,14 @@ if(USE_CUSTOM_LIBCXX)
 endif()
 
 # airtime: start
-## Add minimal googletest targets. The provided one has many side-effects, and
-## googletest has a very straightforward build.
-#add_library(boringssl_gtest third_party/googletest/src/gtest-all.cc)
-#target_include_directories(boringssl_gtest PRIVATE third_party/googletest)
-#
-#include_directories(third_party/googletest/include)
+# Add minimal googletest targets. The provided one has many side-effects, and
+# googletest has a very straightforward build.
+if(ANDROID) # CAR-1705 - enable gtest build for android - required by crypto
+  add_library(boringssl_gtest third_party/googletest/src/gtest-all.cc)
+  target_include_directories(boringssl_gtest PRIVATE third_party/googletest)
+
+  include_directories(third_party/googletest/include)
+endif()
```

Comment:
Now that comment-out was reverted. only airtime change is if(ANDROID) and endif() (and indentation in between).
The comments above 'if' are original.
Can you move "airtime" tag?

Reply:
moved.